packages/core/src/package/SfpPackageBuilder.ts

```diff
@@ -228,6 +232,26 @@ export default class SfpPackageBuilder {
             sfpPackage.diffPackageMetadata = diffPackageInfo;
         }
     }
+
+    private static isAllTestsToBeTriggered(sfpPackage:SfpPackage,logger:Logger) {
+        if (
+            sfpPackage.packageType == PackageType.Source &&
+            sfpPackage.isApexFound == true &&
+            sfpPackage.apexTestClassses == null
```

Comment:
Typo in sfpPackage member `apexTestClasses`

Reply:
Its already an existing member, will refactor in a later PR